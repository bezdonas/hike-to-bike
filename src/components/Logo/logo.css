.logo-block {
  text-align: center;
  font-family: 'Comic Sans MS', cursive, sans-serif;
  font-weight: bold;
  margin: 0;
  font-size: 40px;
  margin-bottom: 40px;
}
