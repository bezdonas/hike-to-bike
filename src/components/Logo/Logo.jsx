import { h, Component } from 'preact';
import './logo.css';

export default class Logo extends Component {
  render() {
    return <h1 className="logo-block">Hike to a&nbsp;bike</h1>;
  }
}
