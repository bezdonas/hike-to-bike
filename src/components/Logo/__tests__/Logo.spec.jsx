import { h, preact } from 'preact';
import render from 'preact-render-to-json';
import Logo from '../Logo.jsx';

describe('Logo', () => {
  it('renders as expected and matches snapshot', () => {
    expect(render(<Logo />)).toMatchSnapshot();
  });
});
