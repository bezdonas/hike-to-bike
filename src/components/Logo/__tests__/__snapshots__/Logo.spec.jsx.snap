// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders as expected and matches snapshot 1`] = `
<h1
  class="logo-block"
>
  Hike to aÂ bike
</h1>
`;
